src/h5web/visualizations/containers/NxSpectrumContainer.tsx

```diff
@@ -35,13 +35,13 @@ function NxSpectrumContainer(props: VisContainerProps): ReactElement {
 
   const { signal, title, errors, axisMapping } = nxData;
 
-  const { setScaleType } = useLineConfig();
+  const { setYScaleType } = useLineConfig();
 
   useEffect(() => {
     if (signal.scaleType) {
-      setScaleType(signal.scaleType);
+      setYScaleType(signal.scaleType);
     }
-  }, [setScaleType, signal.scaleType]);
+  }, [setYScaleType, signal.scaleType]);
```

Comment:
I feel like this `useEffect` should now live in `MappedLineVis`. I know it means adding a prop, but it avoids calling `useLineConfig` in two places.